omas/machine_mappings/d3d.py

```diff
@@ -327,6 +327,92 @@ def pf_active_coil_current_data(ods, pulse):
             data_norm=1.0,
         )
 
+#================================   
+@machine_mapping_function(__regression_arguments__, pulse=170325)
+def ec_launcher_active_hardware(ods, pulse):
+    setup = '.ECH.'
+    # We need three queries in order to retrieve only the fields we need
```

Comment:
the comment about the third query is missing :)
```
# First the amount of systems in use
...
# Second query the used systems to resolve the gyrotron names
...
# Third ???
```

Reply:
I was hoping to get away with 2 queries. Must have forgotten to add that comment when I resigned to use three calls xD